Dockerfile

```diff
@@ -1,13 +1,16 @@
-from python:3.8-alpine
+from python:3.8-slim-buster
```

Comment:
perhaps update this to 3.8.5-slim-buster

Reply:
3.8 references the latest support 3.8 version python. In this case

root@9dd4677296c3:/app# python --version
Python 3.8.6